internal/util/update/update.go

```diff
@@ -124,21 +135,21 @@ func (u Command) Run() error {
 	// require package is checked into git before trying to update it
 	g := gitutil.NewLocalGitRunner(u.Pkg.UniquePath.String())
 	if err := g.Run("status", "-s"); err != nil {
-		return errors.Errorf(
-			"kpt packages must be checked into a git repo before they are updated: %v", err)
+		return fmt.Errorf(
+			"kpt packages must be checked into a git repo before they are updated: %w", err)
 	}
 	if strings.TrimSpace(g.Stdout.String()) != "" {
-		return errors.Errorf("must commit package %s to git before attempting to update",
+		return fmt.Errorf("must commit package %s to git before attempting to update",
 			u.Pkg.UniquePath.String())
 	}
 
 	rootKf, err := u.Pkg.Kptfile()
 	if err != nil {
-		return errors.Errorf("unable to read package Kptfile: %w", err)
```

Comment:
Why this change ? Aren't we using `errors.Errorf` elsewhere?

Reply:
We are in the process of moving to a slightly different way to create errors: https://github.com/GoogleContainerTools/kpt/pull/1639. But those should be addressed in separate PRs, so I have reverted the change here to keep using the `errors.Errorf` package.